### PR TITLE
fix export when using empty ubuntu image

### PR DESCRIPTION
**Pull request by Direnol**

### CODE REVIEW

**Comment by Direnol** [comment]
@GennadySpb

File: post-processor/yandex-export/step-attach-disk.go [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+	yandex.CommonConfig
+	ImageID string
+}
+
+func (c *StepAttachDisk) Run(ctx context.Context, state multistep.StateBag) multistep.StepAction {
+	driver := state.Get("driver").(yandex.Driver)
+	ui := state.Get("ui").(packer.Ui)
+	instanceID := state.Get("instance_id").(string)
+
+	ui.Say("Create disk from source image...")
```
Comment (GennadySpb) [inline post-processor/yandex-export/step-attach-disk.go:23]:
```suggestion
	ui.Say("Create secondary disk from image for export...")
```

File: post-processor/yandex-export/step-attach-disk.go [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+	if op == nil {
+		return yandex.StepHaltWithError(state, err)
+	}
+	protoMD, err := op.Metadata()
+	if err != nil {
+		return yandex.StepHaltWithError(state, err)
+	}
+	md, ok := protoMD.(*compute.CreateDiskMetadata)
+	if !ok {
+		return yandex.StepHaltWithError(state, fmt.Errorf("could not get Instance ID from create operation metadata"))
```
Comment (GennadySpb) [inline post-processor/yandex-export/step-attach-disk.go:52]:
```suggestion
		return yandex.StepHaltWithError(state, fmt.Errorf("could not get Disk ID from create operation metadata"))
```

File: post-processor/yandex-export/step-attach-disk.go [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+	}
+	state.Put("secondary_disk_id", md.GetDiskId())
+
+	if err := op.Wait(ctx); err != nil {
+		return yandex.StepHaltWithError(state, err)
+	}
+
+	ui.Say("Attach disk...")
```
Comment (GennadySpb) [inline post-processor/yandex-export/step-attach-disk.go:60]:
```suggestion
	ui.Say("Attach secondary disk to instance...")
```

**Comment by GennadySpb** [comment]
LGTM!

**Pull request merged**